src/workloads/VSSetup.targets

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Import Project="VSSetup.props" Condition="'$(VSSetupProps)' != '1'"/>
```

Comment:
Why do you need the condition? Does anyone else import this file?

Reply:
Why does the targets file import the props file? Usually the project imports the props and the targets file.